membership/index.md

```diff
@@ -0,0 +1,45 @@
+---
+redirect_from: 
+    - membership/founding-membership
+---
+
+# Become a member
+
+We’re currently looking for members to help shape the Foundation for Public Code’s long term future.
+
+Members are forward-thinking public organizations with a strong commitment to the existence of an ecosystem of public code. As a member you'll help create a global community of public code empowered organizations that pool resources and collaborate in the digital transformation of modern government. This results in higher quality services for the public that are more cost effective, with less risk and more local control.
```

Comment:
"pool resources and collaborate in the digital transformation of modern government" is correct, but in the all cats are animals, but not all animals are cats way. It is correct, but there are many ways this can be filled in. Is this left vague intentionally? Why not clearly state member-owned and member-governed organization that does codebase stewardship?